packages/backend/src/graphql/schema.graphql

```diff
@@ -671,8 +671,10 @@ interface BasePayment {
 type IncomingPayment implements BasePayment & Model {
   "Incoming Payment id"
   id: ID!
-  "Id of the wallet address under which this incoming payment was created"
+  "Id of the wallet address under which this incoming payment was created."
   walletAddressId: ID!
+  "Information about the client resource requesting the incoming payment."
+  client: String
```

Comment:
"Information about the client resource requesting the incoming payment." -> do you think we should explicitly call it "the wallet address of the Open Payments client which created the resource"?

Reply:
Well I can probably be a good guinea pig because I had little/no knowledge of this field...

I think it's just the wallet address right (since its a string)? In which case dropping the "information about" seems clearer to me.